src/data/camera.ts

```diff
@@ -137,6 +137,7 @@ export const getEntityIdFromCameraMediaSource = (mediaContentId: string) =>
 export interface WebRTCClientConfiguration {
   configuration: RTCConfiguration;
   dataChannel?: string;
+  audio_direction: string;
```

Comment:
Following previous attribute names this should be in camelCase to follow code style.
```suggestion
  audioDirection: string;
```

Reply:
Done